.github/workflows/reusable_push_pr.yaml

```diff
@@ -0,0 +1,116 @@
+name: Push/PR no fake prereleases
+
```

Comment:
I would add a few lines of explanation just because we are merging the normal test PR and the test packages creation